### ISSUE TITLE
PartnerCenter.NetCore and Az.KeyVault cannot coexist

### ISSUE THREAD

**Issue by Agazoth** [Microsoft/Partner-Center-PowerShell#70]
# Steps to reproduce

```Powershell
PS C:\> $ClientSecret = Get-AzKeyVaultSecret -VaultName MyKeys -Name MySecret | Select-Object -ExpandProperty SecretValueText
PS C:\> $clientSecretSecure = $clientSecret | ConvertTo-SecureString -AsPlainText -Force
PS C:\> $TenantID = 'MyTenant.onmicrosoft.com'
PS C:\> $Credential = New-Object System.Management.Automation.PSCredential ($ClientAppId, $clientSecretSecure)
PS C:\> $refreshToken = Get-AzKeyVaultSecret -VaultName MyKeys -Name MyRefreshToken | Select-Object -ExpandProperty SecretValueText
PS C:\> $pcToken = New-PartnerAccessToken -RefreshToken $refreshToken -Resource https://api.partnercenter.microsoft.com -Credential $credential
New-PartnerAccessToken : The 'New-PartnerAccessToken' command was found in the module 'PartnerCenter.NetCore', but the module could not be loaded. For more information, run 'Import-Module PartnerCenter.NetCore'.
At line:1 char:12
+ $pcToken = New-PartnerAccessToken -RefreshToken $refreshToken -Resour ...
+            ~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : ObjectNotFound: (New-PartnerAccessToken:String) [], CommandNotFoundException
+ FullyQualifiedErrorId : CouldNotAutoloadMatchingModule

PS C:\>
```

```Powershell
PS C:\> $refreshtoken = 'My very long token'
PS C:\> $ClientAppId = 'My Client Id'
PS C:\> $ClientSecret = 'My Client Secret'
PS C:\> $clientSecretSecure = $clientSecret | ConvertTo-SecureString -AsPlainText -Force
PS C:\> $TenantID = 'MyTenant.onmicrosoft.com'
PS C:\> $Credential = New-Object System.Management.Automation.PSCredential ($ClientAppId, $clientSecretSecure)
PS C:\> $pcToken = New-PartnerAccessToken -RefreshToken $refreshToken -Resource https://api.partnercenter.microsoft.com -Credential $credential
PS C:\> Get-AzKeyVaultSecret -VaultName MyKeys -Name MySecret
Get-AzKeyVaultSecret : The Azure PowerShell session has not been properly initialized.  Please import the module and try again.
At line:1 char:1
+ Get-AzKeyVaultSecret -VaultName MyKeys -Name MySecret
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : NotSpecified: (:) [Get-AzKeyVaultSecret], InvalidOperationException
+ FullyQualifiedErrorId : System.InvalidOperationException,Microsoft.Azure.Commands.KeyVault.GetAzureKeyVaultSecret

PS C:\>
```

## Expected behavior

In the first scenario a refresh token should be issued. The module should be imported automatically. Running Import-Module PartnerCenter.NetCore results in the same error as in the example.

If PartnerCenter.NetCore is imported first, no Az modules can be imported and result in the same error at mentioned above.

## Actual behavior

Error is thrown, stating that the next module cannot be imported.

## Diagnostic logs

Will share if needed

## Environment

> Please share additional details about your environment.
```Powershell
Name             : ConsoleHost
Version          : 6.2.0-preview.3
InstanceId       : 1fac6f49-3aa2-4054-9496-ffe5b6159a84
UI               : System.Management.Automation.Internal.Host.InternalHostUserInterface
CurrentCulture   : en-US
CurrentUICulture : en-US
PrivateData      : Microsoft.PowerShell.ConsoleHost+ConsoleColorProxy
DebuggerEnabled  : True
IsRunspacePushed : False
Runspace         : System.Management.Automation.Runspaces.LocalRunspace

ModuleType Version    Name                                PSEdition ExportedCommands
---------- -------    ----                                --------- ----------------
Script     1.0.1      Az.KeyVault                         Core,Desk {Add-AzKeyVaultCertificate, Update-AzKeyVaultCer...

ModuleType Version    Name                                PSEdition ExportedCommands
---------- -------    ----                                --------- ----------------
Binary     1.5.1902.2 PartnerCenter.NetCore               Core,Desk {Add-PartnerCustomerCartLineItem, Add-PartnerCus...

```

**Comment by isaiahwilliams** [comment]
Hi @Agazoth, 

The root cause for this issue is a version conflict for the Microsoft.Rest.ClientRuntime library. Through my testing if we make it match the version used by the Azure PowerShell module then there is no issue using either command. Before we deploy an update to address this issue, I would like to perform some additional testing to ensure there are not any other items that need to be addressed. Please let me know if you have any discovered any other issues.

**Comment by isaiahwilliams** [comment]
Hi @Agazoth, 

We have released version 1.5.1902.3 for both the PartnerCenter and PartnerCenter.NetCore modules. Once you update to the latest version, this will work as excepted. Please let us know if you encounter any other issues.
